lib/IHP/static/helpers.js

```diff
@@ -42,33 +43,54 @@ function initTime() {
 }
 
 function initDelete() {
-    document.addEventListener('click', handleClick);
+    document.querySelectorAll('.js-delete').forEach(function (elem) {
+        if (elem.hasAttribute('jsDeleteInitialized') === false) {
```

Comment:
![image](https://user-images.githubusercontent.com/2072185/101062976-92d71a80-3592-11eb-9ba4-2980f85d8164.png)

The attribute is visible from the browser dev tools. Can't we also do `elem.jsDeleteInitialized = true`? Should work similiar to the attribute but is invisible. In case that doesn't work we should rename the attribute to something like `data-js-delete-initialized` to be more consistent with HTML attribute conventions

Reply:
`elem.jsDeleteInitialized = true` worked like a charm.